src/input/base-input.js

```diff
@@ -29,6 +29,7 @@ class BaseInput<T: EventTarget> extends React.Component<
     'aria-labelledby': null,
     'aria-describedby': null,
     adjoined: ADJOINED.none,
+    autocomplete: 'on',
```

Comment:
Should we be consistent with `autoFocus` and use `autoComplete` ?

Reply:
sure - I can update that